src/main/kotlin/com/neva/gradle/fork/config/properties/PropertyDefinition.kt

```diff
@@ -0,0 +1,72 @@
+package com.neva.gradle.fork.config.properties
+
+class PropertyDefinitions {
+  private val definitions = mutableMapOf<String, PropertyDefinition>()
+
+  fun configure(propertiesConfiguration: Map<String, PropertyDefinition.() -> Unit>) {
```

Comment:
you are in a scope of `Property` (class name is scoping `Property` keyword) so that why no just `configuration` as an arg name

Reply:
OK